packages/app/src/specs/SpecsListCloudButton.cy.tsx

```diff
@@ -0,0 +1,141 @@
+import { SpecCloudDataHoverButtonFragmentDoc } from '../generated/graphql-test'
+import SpecsListCloudButton from './SpecsListCloudButton.vue'
+import { set } from 'lodash'
+
+describe('<SpecsListCloudButton>', { viewportWidth: 300, viewportHeight: 400 }, () => {
+  function mountWithStatus (
+    status: 'NOT_FOUND' | 'LOGGED_OUT' | 'CONNECTED' | 'NOT_CONNECTED' | 'UNAUTHORIZED' | 'ACCESS_REQUESTED',
+  ) {
+    cy.mountFragment(SpecCloudDataHoverButtonFragmentDoc, {
+      onResult: (result) => {
+        set(result, 'cloudViewer', { __typename: 'CloudUser', id: 'abc123' })
+
+        switch (status) {
```

Comment:
Nice,  very concise :+1:

Reply:
Thanks, but I think I stole it from somewhere else!